documentation/src/docs/recovery-procedures/metadata-table-segment-recovery.md

```diff
@@ -0,0 +1,118 @@
+<!--
+Copyright Pravega Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+# Metadata Table Segment Recovery
+
+
+Please note that the steps outlined below are in continuation to the ones outlined as part of the recovery procedure described [here](https://github.com/pravega/pravega/blob/master/documentation/src/docs/recovery-procedures/table-segment-recovery.md)
+That is we need to perform some of the steps mentioned in the above referenced doc, before continuing any of the steps present here.
+
+The below outlined steps refer to recovery of Table Segments but only in cases of Metadata Segments which are a special case, as they need to 
```

Comment:
This part of the text is similar to what I referred above and it would be good as an introduction for the reader.

Reply:
moved this on top.